app/Console/Commands/ImportSignupsCommand.php

```diff
@@ -70,13 +73,17 @@ public function handle()
                     'northstar_id' => $missing_signup['northstar_id'],
                     'campaign_id' => $missing_signup['campaign_node_id'],
                     'campaign_run_id' => $missing_signup['campaign_run_id'],
-                    'source' => 'sms',
-                    'created_at' => $missing_signup['signup_created_at_timestamp'],
+                    'source' => 'phoenix-next',
```

Comment:
can the `source` be provided in the csv so that we don't have to hard code this?

Reply:
@sbsmith86 yeah it could be! I wasn't sure how future-proof I should make this